### PR TITLE
Add markup for save points

### PR DESCRIPTION
**Pull request by aldreth**
references https://github.com/RaspberryPiFoundation/ups/issues/665

### CODE REVIEW

File: lib/kramdown_rpf/kramdown.rb [hunk 1]
```diff
@@ -152,6 +171,15 @@ def parse_hints
 
       define_parser(:hints, HINTS_PATTERN)
 
+      # Convert Markdown -> :save
+      # @api private
+      def parse_save
+        @src.pos += @src.matched_size
```
Comment (dtt101) [inline lib/kramdown_rpf/kramdown.rb:177]: not familiar with this code - what does this line achieve? (don't say addition!)

Comment (aldreth) [reply lib/kramdown_rpf/kramdown.rb:177]: As I understand it, it allows the parser to keep track of its location in the file. We are adding methods to `Kramdown::Parser::GFM` at this point & you can see similar code in the kramdown gem here https://github.com/gettalong/kramdown/blob/master/lib/kramdown/parser/gfm.rb#L126

Comment (aldreth) [reply lib/kramdown_rpf/kramdown.rb:177]: More info here https://github.com/gettalong/kramdown/blob/master/lib/kramdown/parser/kramdown.rb#L23-L59

Comment (dtt101) [reply lib/kramdown_rpf/kramdown.rb:177]: 👍

**Review by dtt101 — APPROVED** [review]

**Pull request merged**